<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterface</class>
 <widget class="QMainWindow" name="UserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="compressRadio">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprimir</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="uncompressRadio">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Descomprimir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="actionButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Executar</string>
    </property>
   </widget>
   <widget class="QTreeView" name="filesTree">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>501</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="destinyPathLine">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="resourcePathLine">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Origem</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Destino</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
